web/gds-user-ui/src/components/CertificateReview/index.tsx

```diff
@@ -64,11 +65,11 @@ const CertificateReview = () => {
       setIsMainNetSubmitting(false);
       setIsTestNetSubmitting(false);
 
-      if (!err?.response?.data?.success) {
+      if (!err?.data?.success) {
         toast({
           position: 'top-right',
           title: t`Error Submitting Certificate`,
-          description: t`${err?.response?.data?.error}`,
+          description: t`${upperCaseFirstLetter(err?.data?.error)}`,
```

Comment:
This ensures error messages are displayed to users if they have an issue submitting their form.